.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "window.autoDetectColorScheme": true
```

Comment:
I see that adding to gitignore after the file was already added to the PR doesn't remove it,
you have to run git rm --cached .vscode/settings.json
in your vscode's terminal, and then commit & push